src/main/java/com/ververica/flink/table/gateway/operation/OperationUtil.java

```diff
@@ -20,29 +20,32 @@
 
 import com.ververica.flink.table.gateway.rest.result.ColumnInfo;
 import com.ververica.flink.table.gateway.rest.result.ConstantNames;
+import com.ververica.flink.table.gateway.rest.result.ResultKind;
 import com.ververica.flink.table.gateway.rest.result.ResultSet;
 
-import org.apache.flink.table.types.logical.BigIntType;
 import org.apache.flink.table.types.logical.VarCharType;
 import org.apache.flink.types.Row;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 
 /**
  * Utility class for {@link Operation}.
  */
 public class OperationUtil {
 
-	public static final ResultSet AFFECTED_ROW_COUNT0 = new ResultSet(
-		Collections.singletonList(ColumnInfo.create(ConstantNames.AFFECTED_ROW_COUNT, new BigIntType(false))),
-		Collections.singletonList(Row.of(0L)));
+	public static final ResultSet OK = ResultSet.builder()
+		.resultKind(ResultKind.SUCCESS)
+		.columns(ColumnInfo.create(ConstantNames.RESULT, new VarCharType(2)))
+		.data(Row.of("OK"))
```

Comment:
Define `ConstantNames.OK` so that JDBC driver does not need to use a magic string.